### PR TITLE
Model 1 - Day # 2: Simple HTML, & CSS project. 

### PR DESCRIPTION
**Pull request by henrycode460**
This project aim at helping student understand how to use git. It also help with making the understanding the Linters simple.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+
+- GitHub: [@githubhandle](https://github.com/henrycode460/Hello-World/)
+- 
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (topeogunleye) [inline README.md:28]: - [ ] [OPTIONAL] I kindly suggest that you input your project GitHub issues link into the parenthesis 

### EXAMPLE
`[issues page](https://github.com/henrycode460/Hello-World/issues)`

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+- css
+
+
+
+
+
+
+## Authors
+
+👤 **Author1**
```
Comment (topeogunleye) [inline README.md]: - [ ] [OPTIONAL] Kindly add your name in place of Author1

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @henrycode460,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> 
- [ ] *Kindly make sure your linters are all passing before submitting the PR.
> 
> ## [Optional] suggestions:
> - [ ] Kindly make use of imperative mood in the subject line for your commit messages. 

> Example – Add fix for dark mode toggle state
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+h1 {
```
Comment (topeogunleye) [inline style.css]: - [ ] [REQUIRED] You are using an element selector to style the heading, I kindly suggest using a class selector instead to follow the project requirements. You can add to class to the heading and use that class here to style the heading.

**Review by OLIPLICHE — APPROVED** [review]
Hi @henrycode460 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**